src/client.ts

```diff
@@ -101,30 +130,26 @@ export interface IPageData {
     title: string;
 }
 
-/** One page entry in a layout tree. */
-export interface ITreeEntry {
-    depth: number;
-    route: string;
+/*
+LAYOUT HIERARCHY NODES
```

Comment:
hmm these are all pretty intertwined and it's nice having all the symbols in one file.